format_example.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "dfa65597-eb1b-4127-9200-e53cc41d2d93",
+  "name": "Example Project",
+  "summary": "This is a short, one-sentence summary of the Example Project.",
+  "icon": "https://example.com/example.png",
+  "description": "This is a long description describing the project. <b>HTML</b> and **Markdown** can optionally be used.",
+  "gallery": [
+    {
+      "type": "video",
+      "embed": "https://youtube.com/embed...",
+      "caption": "This is an example embed."
+    },
+    {
+      "type": "video",
+      "src": "https://example.com/video.mp4",
+      "caption": "This is an example direct video link."
+    },
+    {
+      "type": "image",
+      "src": "https://example.com/image.png",
+      "caption": "This is an example image."
+    }
+  ],
+  "authors": [
+    {
+      "name": "MCIP Contributors",
+      "uri": "https://github.com/mc-cip"
+    }
+  ],
+  "categories": [
+    {
+      "name": "Example Category",
+      "uri": "https://example.com/browse?category=example",
+      "icon": "https://example.com/example-category.png"
```

Comment:
This is a bit pedantic, but the category icon could have a checksum.

Reply:
I'm not sure if category information should even be a mod spec file. Just a list of categories should be all that's needed from a mod's spec

Reply:
It probably is redundant to include category icons and URIs in each mod - I may replace it with a list of category names, and instead have a map between category names and icons/URIs returned separately.